### PR TITLE
Update project and template jshintrc

### PR DESCRIPTION
**Pull request by robwierzbowski**
This updates the project template `jshintrc` to the same standards as the repo `.jshintrc`. Also removes the now default `es5` rule.

### CODE REVIEW

**Comment by passy** [comment]
Isn't `browser` still useful for the generated app?

**Comment by robwierzbowski** [comment]
Absolutely, missed that. Amended.

File: app/templates/jshintrc [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
+  "eqeqeq": true,
+  "eqnull": true,
+  "immed": true,
+  "latedef": true,
+  "newcap": true,
+  "noarg": true,
+  "undef": true,
+  "strict": false,
+  "trailing": true,
+  "smarttabs": true
```
Comment (sindresorhus) [inline app/templates/jshintrc]: You can remove the `es5`, but the rest has been carefully chosen and should be as it is. Also don't change indentation on PRs.

**Comment by robwierzbowski** [comment]
Ok, I can revise. Although I'm unclear why you would enforce camelcase, indentation, and single quotes in the generator and not the project. The indentation change was to match the generator's .jshintrc indentation.

**Comment by passy** [comment]
Thanks!

We just have different code styles for both. The editorconfig looks different for generator and project, too.

**Comment by robwierzbowski** [comment]
Interessant, I'll take a look. Thanks!

**Comment by sindresorhus** [comment]
@robwierzbowski the project jshint files could be updated to the generator one except for the indentation which differs. PR welcome, but has to contain code changes so it passes JSHint ;)

**Pull request merged**